impeller/display_list/skia_conversions_unittests.cc

```diff
@@ -180,8 +180,33 @@ TEST(SkiaConversionsTest, IsNearlySimpleRRect) {
       SkRRect::MakeRectXY(SkRect::MakeLTRB(0, 0, 10, 10), 10, 10)));
   EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(
       SkRRect::MakeRectXY(SkRect::MakeLTRB(0, 0, 10, 10), 10, 9.999)));
-  EXPECT_FALSE(skia_conversions::IsNearlySimpleRRect(
+  EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(
       SkRRect::MakeRectXY(SkRect::MakeLTRB(0, 0, 10, 10), 10, 9)));
+  EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(
+      SkRRect::MakeRectXY(SkRect::MakeLTRB(0, 0, 10, 10), 10, 5)));
+  SkRect rect = SkRect::MakeLTRB(0, 0, 10, 10);
+  SkRRect rrect;
+  union {
+    SkPoint radii[4] = {
+        {10.0f, 9.0f},
+        {10.0f, 9.0f},
+        {10.0f, 9.0f},
+        {10.0f, 9.0f},
+    };
+    SkScalar values[8];
+  } test;
+  rrect.setRectRadii(rect, test.radii);
+  EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(rrect));
+  for (int i = 0; i < 8; i++) {
+    auto save = test.values[i];
+    test.values[i] -= kEhCloseEnough * 0.5f;
+    rrect.setRectRadii(rect, test.radii);
+    EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(rrect));
```

Comment:
Since this is in a loop the line number isn't sufficient to know where the failure happened.  Can you also print out the radii?


something like:
```suggestion
    EXPECT_TRUE(skia_conversions::IsNearlySimpleRRect(rrect)) << "radii: " << test.radii;
```

Reply:
I don't think it will print the array but I can print the index and the substituted value...

Reply:
Yeah, the array just prints out the address, but I can do this (temporarily sabotaging the test to get error output):

```
../../flutter/impeller/display_list/skia_conversions_unittests.cc:204: Failure
Value of: skia_conversions::IsNearlySimpleRRect(rrect)
  Actual: true
Expected: false
values[7] == 8.9995002746582031

../../flutter/impeller/display_list/skia_conversions_unittests.cc:208: Failure
Value of: skia_conversions::IsNearlySimpleRRect(rrect)
  Actual: false
Expected: true
values[7] == 8.9975004196166992
```